src/index.d.ts

```diff
@@ -28,6 +28,7 @@ declare namespace preact {
 		 * Default value: `-1`
 		 */
 		endTime?: number;
+		_: VNode;
```

Comment:
No other private properties are documented here. Is there a reason to document this one?